packages/node_modules/@webex/plugin-meetings/src/locus-info/infoUtils.js

```diff
@@ -47,9 +47,17 @@ InfoUtils.parseDisplayHintSection = (info, displayHintKey) => {
 
 InfoUtils.parsePolicy = (info) => InfoUtils.parseDisplayHintSection(info, 'joined');
 
-InfoUtils.parseModerator = (info) => InfoUtils.parseDisplayHintSection(info, 'moderator');
+InfoUtils.parseModerator = (info) => {
+  const displayHints = InfoUtils.parseDisplayHintSection(info, 'moderator');
 
-InfoUtils.parseCoHost = (info) => InfoUtils.parseDisplayHintSection(info, 'coHost');
+  return {...displayHints, [DISPLAY_HINTS.LOWER_SOMEONE_ELSES_HAND]: true};
+};
+
+InfoUtils.parseCoHost = (info) => {
+  const displayHints = InfoUtils.parseDisplayHintSection(info, 'coHost');
+
+  return {...displayHints, [DISPLAY_HINTS.LOWER_SOMEONE_ELSES_HAND]: true};
+};
```

Comment:
nitpick: Similarly this could have been:

```js
InfoUtils.parseCoHost = (info) => ({
  [DISPLAY_HINTS.LOWER_SOMEONE_ELSES_HAND]: true,
  ...InfoUtils.parseDisplayHintSection(info, 'coHost'),
});
```